antarest/study/business/areas/hydro_management.py

```diff
@@ -110,18 +111,42 @@ class ManagementOptionsFormFields(FormFieldsBaseModel):
 class HydroManager:
     def __init__(self, command_context: CommandContext) -> None:
         self._command_context = command_context
+        self.area_id = ""
+
+    @staticmethod
+    def get_id(area_id: str, field_dict: Dict[str, FieldInfo]) -> str:
+        """
+        Try to match the current area_id with the one from the original file.
+        These two ids could mismatch based on their character cases since the id from
+        the filesystem could have been modified with capital letters.
+        We first convert it into lower case in order to compare both ids.
+
+        Returns the area id from the file if both values matched, the initial area id otherwise.
+        """
+        for file_area_id in field_dict:
+            if LowerCaseIdentifier.generate_id(file_area_id) == area_id:
+                return file_area_id
+        return area_id
+
+    @staticmethod
+    def get_hydro_config(study: StudyInterface) -> Dict[str, Dict[str, FieldInfo]]:
```

Comment:
You can add a underscore in front of the method name to make it private, as you only use it in this class